database/postgresql/postgresql_ext.py

```diff
@@ -175,6 +175,8 @@ def main():
 
         elif state == "present":
             changed = ext_create(cursor, ext)
+    except SystemExit, e:
+        raise
```

Comment:
I'm not sure what you are trying to do here, it seems like a noop and also we try to avoid returning exceptions from modules